tests/examples/tokens/test_erc1155.py

```diff
@@ -160,6 +162,11 @@ def test_URI(erc1155, w3, assert_tx_failed):
 
     assert_tx_failed(lambda: erc1155.setURI(CONTRACT_URI))
 
+    # set contract to dynamic URI
+    erc1155.toggleDynUri(True, transact={"from": owner})
+    erc1155.setURI(CONTRACT_DYNURI, transact={"from": owner})
+    assert erc1155.uri(0) == CONTRACT_DYNURI + 0
```

Comment:
should this be `str(0)`?

Reply:
Most likely. I will take care of this tonight.

Reply:
Taken care of it. 
See some tests fail now with no output on the "tox" command.